programs/uxd/tests/integration_tests/suites/test_mint_and_redeem.rs

```diff
@@ -202,44 +200,12 @@ async fn test_mint_and_redeem() -> Result<(), program_test_context::ProgramTestE
     .await
     .is_err());
 
-    // Find the important PDAs to resolve the depositories address to be whitelisted
-    let identity_depository = program_uxd::accounts::find_identity_depository_pda().0;
-
-    let mercurial_base = program_mercurial::accounts::find_base();
-    let mercurial_vault = program_mercurial::accounts::find_vault_pda(
-        &collateral_mint.pubkey(),
-        &mercurial_base.pubkey(),
-    )
-    .0;
-    let mercurial_vault_depository = program_uxd::accounts::find_mercurial_vault_depository_pda(
-        &collateral_mint.pubkey(),
-        &mercurial_vault,
-    )
-    .0;
-
-    let credix_market_seeds = program_credix::accounts::find_market_seeds();
-    let credix_global_market_state =
-        program_credix::accounts::find_global_market_state_pda(&credix_market_seeds).0;
-    let credix_lp_depository = program_uxd::accounts::find_credix_lp_depository_pda(
-        &collateral_mint.pubkey(),
-        &credix_global_market_state,
-    )
-    .0;
-
-    // Set the controller's depositories addresses
-    program_uxd::instructions::process_edit_controller(
+    // Now we set the router depositories to the correct PDAs
+    program_uxd::procedures::process_set_router_depositories(
```

Comment:
moving this code to this helper function, so that it can be called in other test suites easily